src/pages/issues/index.tsx

```diff
@@ -47,6 +66,17 @@ const SearchField = ({ onSearchTextSubmitted, loading }: SearchFieldProps) => {
 };
 
 const Issues: FC = () => {
+    let splittedQueryParam = '';
+    const router = useRouter();
+    const dev = router?.query?.dev;
+    if (dev === 'true') {
+        const urlQueryParam = router?.query?.q as string;
+        if (urlQueryParam) {
+            const splittedQueryParamArr = urlQueryParam.split(':');
+            splittedQueryParam = splittedQueryParamArr[1];
```

Comment:
These can be part of a util function.

Reply:
Yes, created a utility function for the same. Thanks.